replicator-resources.qmd

```diff
@@ -309,6 +309,18 @@ You can try the web-based uploader which will take any URL directly:
 
 ![](img/nuvolos-large-upload.png)
 
+When copying the dropbox URL, remember to change 'dl=0' to 'dl=1' inside the URL. This forces the download. For example:
+```
+https://www.dropbox.com/scl/fo/gumryf1zs8lwk5zq5udo7/ABrnRWk5w859IIibD-_QM1Q/Szumilo-20230708-R1?*dl=1*&preview=Niko+Szumilo+-+MS20230708-replication.zip&rlkey=ei2aozcysqcchaoiwwanjcg4s&subfolder_nav_tracking=1
+```
+
+To rename the file, open the terminal and write
+```
+mv 'old_file_name' 'new_file_name'
+rm 'old_file_name'
+```
+
+If the file is very large, move it to the large file storage that you can create in the 'Project Configuration' tab.
```

Comment:
this part is actually pretty hard to read what I did here. could you maybe change this bit into a more readable example please?  not much to change but remove the `(base)` stuff and shorten the super long name a bit (maybe refer to tab completion again - people are not suppose to type that thing...) thank